proc_gradient.py

```diff
@@ -1,164 +0,0 @@
-from pyspecdata import *
```

Comment:
So I wasn't sure if you wanted to keep scripts which I did not create in this repo. This here is Alec's processing. I have never performed anything with gradients. I can add this back in but I definitely am not up to speed on what's going on in this script since I haven't performed this type of experiment before.